mobile/lib/modules/home/ui/asset_grid/immich_asset_grid_view.dart

```diff
@@ -237,6 +263,9 @@ class ImmichAssetGridViewState extends State<ImmichAssetGridView> {
   void initState() {
     super.initState();
     scrollToTopNotifierProvider.addListener(_scrollToTop);
+    scrollToDateNotifierProvider
```

Comment:
Do you need to remove this listener on dispose?

Reply:
Oh yeah, thanks!